WISE-Community/WISE
# Discussion gating authoring option

This commit allows authors to allow/disallow students to seeing other students' posts before creating new post in discussion component. The JSON field is called ```gateClassmateResponses```.

Test that you can 
- enable/disable the ```gateClassmateResponses``` option in the discussion component authoring
- see others' posts without creating a new post if gating is disabled (```gateClassmateResponses=false```)
- NOT see others' posts without creating a new post if gating is enabled (```gateClassmateResponses=true```)
